libraries/rodeos/include/b1/rodeos/filter.hpp

```diff
@@ -17,8 +22,33 @@ struct callbacks;
 using rhf_t     = registered_host_functions<callbacks>;
 using backend_t = eosio::vm::backend<rhf_t, eosio::vm::jit>;
 
+#ifdef EOSIO_EOS_VM_OC_RUNTIME_ENABLED
+struct eosvmoc_tier {
+   eosvmoc_tier(const boost::filesystem::path& d, const eosio::chain::eosvmoc::config& c,
+                const std::vector<uint8_t>& code, const eosio::chain::digest_type& code_hash)
+       : cc(d, c,
+            [code, code_hash](const eosio::chain::digest_type& id, uint8_t vm_version) -> std::string_view {
+               if (id == code_hash)
+                  return { reinterpret_cast<const char*>(code.data()), code.size() };
+               else
+                  return {};
+            }),
+         exec(cc), mem(512, get_intrinsic_map()), hash(code_hash) {
```

Comment:
Filters currently run with eos-vm's current default max pages (65536)

Reply:
That's fine.  The limit here is the threshold at which the implementation switches to `mprotect`, not the maximum number of pages.

Reply:
It does need to be fixed in `rodeos.cpp`